tests/ui/layout/hexagon-enum.stderr

```diff
@@ -61,11 +61,13 @@ error: layout_of(A) = Layout {
                        },
                        max_repr_align: None,
                        unadjusted_abi_align: Align(1 bytes),
+                       randomization_seed: 2476547119765241672,
```

Comment:
Question: however, this is not normalized out. Wouldn't this be quite sensitive and have to be re-blessed by seeming unrelated PRs?

Reply:
Yeah, this should definitively be normalized away.

Reply:
Fixed.